### PR TITLE
Tf fixes

### PR DESCRIPTION
**Pull request by tarasborov**
The urdf file is correct.

### CODE REVIEW

File: rs_description/frames.gv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+"body" -> "front_rail"[label="Broadcaster: /robot_state_publisher\nAverage rate: 10000.000 Hz\nMost recent transform: 0.000 ( 11.153 sec old)\nBuffer length: 0.000 sec\n"];
+"body" -> "rear_rail"[label="Broadcaster: /robot_state_publisher\nAverage rate: 10000.000 Hz\nMost recent transform: 0.000 ( 11.153 sec old)\nBuffer length: 0.000 sec\n"];
+"body" -> "front_left_hip"[label="Broadcaster: /robot_state_publisher\nAverage rate: 10.220 Hz\nMost recent transform: 11.100 ( 0.053 sec old)\nBuffer length: 4.599 sec\n"];
+"front_left_hip" -> "front_left_upper_leg"[label="Broadcaster: /robot_state_publisher\nAverage rate: 10.220 Hz\nMost recent transform: 11.100 ( 0.053 sec old)\nBuffer length: 4.599 sec\n"];
+"front_left_upper_leg" -> "front_left_lower_leg"[label="Broadcaster: /robot_state_publisher\nAverage rate: 10.220 Hz\nMost recent transform: 11.100 ( 0.053 sec old)\nBuffer length: 4.599 sec\n"];
+edge [style=invis];
+ subgraph cluster_legend { style=bold; color=black; label ="view_frames Result";
+"Recorded at time: 11.153"[ shape=plaintext ] ;
+ }->"body";
+}
```
Comment (ldemk) [inline rs_description/frames.gv]: What is the purpose of this file?

Comment (ldemk) [reply rs_description/frames.gv]: The file frames.pdf should be removed from commit

Comment (ldemk) [reply rs_description/frames.gv]: The file frames.pdf should be removed from commit

File: rs_description/package.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rs_description</name>
+  <version>0.0.0</version>
+  <description>The spot_description package</description>
+
+  <maintainer email="dniewinski@clearpathrobotics.com">Dave Niewinski</maintainer>
```
Comment (ldemk) [inline rs_description/package.xml]: We have made significant changes you can change the maintainer

File: rs_description/package.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>robot_state_publisher</exec_depend>
+  <exec_depend>urdf</exec_depend>
+  <exec_depend>xacro</exec_depend>
+
+  <export>
+  </export>
+</package>
```
Comment (ldemk) [inline rs_description/package.xml:19]: rs_description/urdf/.spot.urdf.xacro.swp  has to be removed. I believe.

File: rs_gazebo/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+  CATKIN_DEPENDS gazebo_ros gazebo_ros_control rs_description
+)
+
+install(FILES
+  launch/gazebo.launch
+  worlds/office.world
```
Comment (ldemk) [inline rs_gazebo/CMakeLists.txt]: which file re we running? Is it reallly office.world?

**Review by ldemk — CHANGES_REQUESTED** [review]

**Comment by ldemk** [comment]
@tarasborov remove all files that does not belong to HQ world, like the following
 rs_gazebo/cad/ss_austin/long_table_collision.blend 

Please, leave **only** HQ world in Lviv (the 4th floor)

File: rs_description/frames.gv (hunk 1 above)
Comment (tarasborov) [reply rs_description/frames.gv]: > What is the purpose of this file?

It is created by tf tree viewer. It is deleted in ab2d03dcc65d68858d7bc52654391fd2d3d36552

File: rs_gazebo/CMakeLists.txt (hunk 4 above)
Comment (tarasborov) [reply rs_gazebo/CMakeLists.txt]: It is a typo. Changed in  ab2d03dcc65d68858d7bc52654391fd2d3d36552

**Review by ldemk — APPROVED** [review]

**Pull request merged**